core/schema/llm.go

```diff
@@ -173,8 +173,9 @@ func (s *llmSchema) loop(ctx context.Context, llm *core.LLM, args struct{}) (*co
 func (s *llmSchema) attempt(_ context.Context, llm *core.LLM, _ struct {
 	Number int
 }) (*core.LLM, error) {
-	// nothing to do; we've "forked" it by nature of changing the query
-	return llm, nil
+	// clone the LLM object, since it updates in-place when Sync is called, and we
+	// want to branch off from here
+	return llm.Clone(), nil
```

Comment:
This is a subtle fix to make this consistent with the recent refactor to how `sync` works: it updates in-place to ensure all unlaziers (`tokenUsage`, `lastReply`, etc) are called against the same result.

As-is, if you append `Attempt(1..10)` and then call `LastReply` it'll actually only evaluate once. So the fix is to just clone it.

Seems like kind of a pain to test. :thinking: Open to ideas. I noticed this by looking through the evals traces. It doesn't affect evals on `main` because they call `Attempt` before appending their prompt.